### PR TITLE
leaking-vars-tests

### PR DESCRIPTION
**Pull request by ArmorDarks**
Added requested in #9 files

I hope I didn't miss anything.

Should I update `compare.js`, or create new one in `tests/leaking-vars`?

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -21,7 +21,10 @@ module.exports = function(grunt) {
       },
       render: {
         files: {
-          'tests/_output.html' : ['tests/input.html']
+          'tests/_output.html' : ['tests/input.html'],
+          'tests/_output.html' : ['tests/input.html'],
```
Comment (vitkarpov) [inline Gruntfile.js]: seems it's the same as above

**Comment by vitkarpov** [comment]
I think we could use the same `compare.js`, at least for a while

**Comment by vitkarpov** [comment]
It would be great to see test in `compare.js` which fails, so I could merge the pr and try to explore the problem. The solution is when the test starts to pass.

**Comment by ArmorDarks** [comment]
Ops, fixed doubled line :)

I've added `compare.js`, but I'm not sure about it's syntax, since I'm not very familiar with expect. Please, check it out.

File: tests/leaking-vars/compare.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+var expect = require('expect.js');
+var fs = require('fs');
+var util = require('util');
+
+var sample1 = fs.readFileSync('tests/leaking-vars/output1.html').toString();
+var sample2 = fs.readFileSync('tests/leaking-vars/output1.html').toString();
+var test1 = fs.readFileSync('tests/leaking-vars/_output2.html').toString();
+var test2 = fs.readFileSync('tests/leaking-vars/_output2.html').toString();
```
Comment (vitkarpov) [inline tests/leaking-vars/compare.js]: we need to compare `output1` width `_output1` and `output2` with `_output2`.

```
var sample1 = fs.readFileSync('tests/leaking-vars/output1.html').toString();
var sample2 = fs.readFileSync('tests/leaking-vars/output2.html').toString();
var test1 = fs.readFileSync('tests/leaking-vars/_output1.html').toString();
var test2 = fs.readFileSync('tests/leaking-vars/_output2.html').toString();
```

?

**Comment by vitkarpov** [comment]
Seems ok :+1: Let's fix the comparing as I mentioned above and let's merge this.

**Comment by ArmorDarks** [comment]
Uh, I'd want to say that I'm just still learnin js, but it seems to be more related to my inattentiveness :(

I hope that time I didn't miss anything.

**Pull request merged**